core/src/test/java/io/questdb/test/griffin/engine/window/WindowFunctionUnitTest.java

```diff
@@ -557,20 +701,20 @@ public void testMinRangeUnbounded() {
                 TestDefaults.createMemoryCARW(),
                 TestDefaults.createMemoryCARW(),
                 0,
-                MinDoubleWindowFunctionFactory.LESS_THAN,
-                MinDoubleWindowFunctionFactory.NAME
+                MaxDoubleWindowFunctionFactory.GREATER_THAN,
+                MaxDoubleWindowFunctionFactory.NAME
         );
         long a = -1930193130;
         long b = -1137976524;
         long c = -1137976524;
         f.computeNext(TestDefaults.createRecord(columnTypes, (long) 46, 19, a));
         f.computeNext(TestDefaults.createRecord(columnTypes, (long) 119, 19, b));
         f.computeNext(TestDefaults.createRecord(columnTypes, (long) 200, 19, c));
-        Assert.assertEquals(f.getDouble(null), a, 1e-6);
+        Assert.assertEquals(f.getDouble(null), b, 1e-6);
```

Comment:
there are a few warnings about flipped expected and actual values

Reply:
fixed it